packages/eslint-plugin/src/configs/all.ts

```diff
@@ -4,6 +4,6 @@
 export default {
   plugins: ['@react-three'],
   rules: {
-    '@react-three/no-clone-in-frame-loop': 'error',
+    '@react-three/no-new-in-loop': 'error',
```

Comment:
I thought this rule name was more succinct, and instead of putting vector and new object violations in a single rule they can be two separate rules.